### ISSUE TITLE
ArrayIndexOutOfBoundsException when merging to prefix blocks cidrs 0.0.0.0/1 or 128.0.0.0/1

### ISSUE THREAD

**Issue by philippe-perrault** [seancfoley/IPAddress#32]
If cidrs "128.0.0.0/1" or "0.0.0.0/1" are merged together in any combinations, ```mergeToPrefixBlocks``` throws exception.

```java
var first = new IPAddressString("128.0.0.0/1");
var second = new IPAddressString("128.0.0.0/1");
first.getAddress().mergeToPrefixBlocks(second.getAddress());
```

**Comment by seancfoley** [comment]
Yes, confirmed, it's an issue with those two cidrs you listed. Thanks for reporting.

**Comment by seancfoley** [comment]
This is fixed in master, the lines 1595 to 1602 in this file of this commit: https://github.com/seancfoley/IPAddress/commit/1e095b20dff6868a51d94f2a35d6f31c2d83d61c#diff-df7ae6b4a6144953e0e2bdbd50763672R1595
There will be a new release with this fix soon.

**Comment by seancfoley** [comment]
This is fixed in versions 4.3.3 and 5.2.0
